test/ng/qSpec.js

```diff
@@ -210,7 +209,7 @@ describe('q', function() {
       });
     };
 
-    afterEach(function() {
+    beforeEach(function() {
```

Comment:
Doesn't this mean that we don't end up cleaning up after the tests?
If we need these values to be `null` instead of `undefined`, we could leave this line as `afterEach` and then just initialize the variables where they are declared above.